tests/test_kem.c

```diff
@@ -127,6 +132,37 @@ static OQS_STATUS kem_test_correctness(const char *method_name) {
 		printf("shared secrets are equal\n");
 	}
 
+	if ((0 == strcasecmp(method_name, "ML-KEM-512")) || (0 == strcasecmp(method_name, "ML-KEM-768")) || (0 == strcasecmp(method_name, "ML-KEM-1024"))) {
+		// buffer to hold z and c. z is always 32 bytes
+		uint8_t *buff_z_c = NULL;
+		size_t length_z_c = (size_t)32 + kem->length_ciphertext;
+		buff_z_c = OQS_MEM_malloc(length_z_c) ;
+		if (NULL == buff_z_c) {
+			fprintf(stderr, "ERROR: OQS_MEM_malloc failed\n");
+			goto err;
+		}
+		// test rejection key by corrupting the secret key
+		secret_key[0] += 1;
+		uint8_t shared_secret_r[32]; // expected output
```

Comment:
Could the size refer to a macro? E.g. `OQS_KEM_ml_kem_512_length_shared_secret`, to avoid hardcoding constants.

Reply:
done. macro added instead of 32

Reply:
Thanks!